docs/src/main/sphinx/installation/containers.md

```diff
@@ -88,6 +88,18 @@ at `/etc/trino/catalog`, or individual catalog property files in it.
 
 If you want to use additional plugins, mount them at `/usr/lib/trino/plugin`.
 
+To avoid having to create catalog files and mount them in the container,
+dynamic catalog management can be enabled by setting the `CATALOG_MANAGEMENT`
+environmental variable to `dynamic`.
+
+```text
```

Comment:
I think shell or bash is better

```suggestion
```shell
```

Reply:
Should we remove the prompt character after switching to `shell`? I think the text should only contain valid commands.

Reply:
I left it for now .. not too critical